### ISSUE TITLE
Install 3.1.0 fails on CYPRESS_BINARY_VERSION being set

### ISSUE THREAD

**Issue by Athaphian** [cypress-io/cypress#2623]
### Current behavior:

When both old AND new env variables are set:
export CYPRESS_BINARY_VERSION=/.../cypress142.zip
export CYPRESS_INSTALL_BINARY=/.../cypress310.zip

Running yarn install on a clean project that depend on Cypress 3.1.0 results in an error:

```
node_modules/cypress: Command failed.
Exit code: 1
Command: node index.js --exec install
Arguments: 
Directory: /.../node_modules/cypress
Output:
The environment variable CYPRESS_BINARY_VERSION has been renamed to CYPRESS_INSTALL_BINARY as of version 3.0.0

You should set CYPRESS_INSTALL_BINARY instead.
----------
```

### Desired behavior:

Install should just look if the new variable CYPRESS_INSTALL_BINARY is present and not fail if the old one is also present.

In our build environment we depend on base docker images that specify the CYPRESS_BINARY_VERSION environment variable. In our own Dockerfile we are unable to UNSET this env variable since docker does not support unsetting variables. This prevents us from upgrading Cypress to a new version..

### Steps to reproduce:

See Current behavior.

### Versions

Version 3.1.0.

**Comment by Bkucera** [comment]
@Athaphian good idea. Maybe just make it a warning, and not fail the install? Or would you prefer failing if `CYPRESS_INSTALL_BINARY` isn't set

**Comment by Bkucera** [comment]
@Athaphian CYPRESS_INSTALL_BINARY isn't required. We basically renamed CYPRESS_BINARY_VERSION to CYPRESS_INSTALL_BINARY. 

You can unset environment variables by overwriting them to an empty string

**Comment by Athaphian** [comment]
Yes thanks! We already found out that setting the env variable to empty works in our docker image. So for now we have a workaround :). Would still be better not to fail the process. That would save other people this semi-painful migration.
